python/lsst/meas/deblender/plugins.py

```diff
@@ -32,12 +32,11 @@
 
 
 def clipFootprintToNonzeroImpl(foot, image):
-    '''
-     Clips the given *Footprint* to the region in the *Image*
+    """Clips the given *Footprint* to the region in the *Image*
```

Comment:
Remember that the summary [has to be one sentence](https://developer.lsst.io/python/numpydoc.html#short-summary): this is too long.